test/modules/project.service.js

```diff
@@ -16,178 +16,108 @@ const git = require('simple-git/promise');
 const fs = require('fs-extra');
 const path = require('path');
 const zlib = require('zlib');
+const klawSync = require('klaw-sync');
```

Comment:
I can't see a new entry in package.json for this? Is it already a dependancy of another package?

Reply:
I just moved this here from the original `remote-bind.test.js`